### DISCUSSION TITLE
Highlight a word without selection

### DISCUSSION THREAD

**Discussion by balbinott** [ueberdosis/tiptap#2214] [General Questions] [answered]
Hi, I'm trying to highlight a word without preselection, but I don't want to select it via code, set highlighting, deselect... is there any way to do this using only range or something like that ?
Thanks!

**Comment by Priestch** [comment, marked as answer]
@balbinott The selection just give you a certain range for highlight, so if you already know the range, you can use transaction to a create [decorations](https://prosemirror.net/docs/ref/#view.Decorations).
